test/Streamly/Test/Data/Unfold.hs

```diff
@@ -449,6 +450,14 @@ enumerateFromToFractional =
 -- Stream transformation
 -------------------------------------------------------------------------------
 
+scan :: Property
+scan =
+    property
+        $ \(ls :: [Int]) ->
+              let unf = UF.scan Fold.sum UF.fromList
+                  mList = scanl1 (+) ls
```

Comment:
Looks like `UF.scan` should be renamed to `UF.postscan`.

Also, I noticed that `UF.scanlM'` is marked as `Unimplemented` even though it is implemented. We can fix that too. Also. UF.scanlM' should be renamed as `postscanlM'`.

Also, rename this test as `postscan`.